src/extends/room/meta.js

```diff
@@ -164,7 +164,13 @@ Room.isHallway = function (name) {
 }
 
 Room.isClaimable = function (name) {
-  if (!Game.map.isRoomAvailable(name)) {
+  if (Game.gcl.level >= GCL_NOVICE && Game.map.getRoomStatus(name).status === 'novice') {
+    return false
+  }
+  if (Game.gcl.level < GCL_NOVICE && Game.map.getRoomStatus(name).status === 'normal') {
```

Comment:
This condition will prevent Quorum from expanding outside novice and respawn areas if its GCL is less than GCL_NOVICE.